### PR TITLE
Fix/clean configurations

### PR DESCRIPTION
**Pull request by Odrakir**
This is my last attempt at cleaning the configurations so everything (Rider, VS, cake, bitrise) is happy... Rider still complains about something, but it's related to the `Release.AppStore` config, so I'm ok with that.

### CODE REVIEW

File: Toggl.Daneel/Toggl.Daneel.csproj [hunk 1]
```diff
@@ -65,41 +65,21 @@
...
-    <MtouchVerbosity>
-    </MtouchVerbosity>
+    <MtouchVerbosity></MtouchVerbosity>
     <MtouchFloat32>true</MtouchFloat32>
     <CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
     <MtouchLink>Full</MtouchLink>
     <MtouchArch>ARMv7, ARM64</MtouchArch>
     <MtouchHttpClientHandler>NSUrlSessionHandler</MtouchHttpClientHandler>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release.UITest|iPhoneSimulator' ">
```
Comment (willsb) [inline Toggl.Daneel/Toggl.Daneel.csproj]: How do you plan on running UITests without this config?

File: Toggl.Ultrawave.Tests/Toggl.Ultrawave.Tests.csproj [hunk 2]
```diff
@@ -12,9 +12,6 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <OutputPath>..\bin\Debug\</OutputPath>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
```
Comment (willsb) [inline Toggl.Ultrawave.Tests/Toggl.Ultrawave.Tests.csproj]: Do you plan on releasing those libs in debug mode?

Comment (Odrakir) [reply Toggl.Ultrawave.Tests/Toggl.Ultrawave.Tests.csproj]: Those are Tests. Do we really need Release config there?

File: Toggl.Daneel/Toggl.Daneel.csproj (hunk 1 above)
Comment (Odrakir) [reply Toggl.Daneel/Toggl.Daneel.csproj]: With the Debug config?

**Comment by willsb** [comment]
Please merge this with a more appropriate name than the PR description, please

**Pull request merged**